samples/mysql-schema/src/main/resources/application.properties

```diff
@@ -1,6 +1,7 @@
 quarkus.operator-sdk.crd.apply=true
-quarkus.operator-sdk.generate-csv=true
 quarkus.container-image.builder=jib
 quarkus.native.additional-build-args=--initialize-at-run-time=org.apache.commons.lang3.RandomStringUtils
 
 quarkus.datasource.username=root
+quarkus.operator-sdk.bundle.channels=alpha
```

Comment:
Are these required? Maybe there should be default values?

Reply:
This is a good question. Yes, the channels and package properties are required, but I'm not sure if providing default values is the right approach. But, as you said, let's put some defaults and let's see how users use it, so:
What about if we use the application name as package name by default?
For the channels, let's use just "alpha" by default.
Wdyt?

Reply:
That seems reasonable indeed.

Reply:
Updated